Sources/Tracking/Service/HttpClient.swift

```diff
@@ -127,4 +128,15 @@ extension CIOHttpClient {
 
         return encodedRawHeader.base64EncodedString(options: NSData.Base64EncodingOptions(rawValue: 0))
     }
+    
+    static func getUserAgent() -> String {
```

Comment:
Can you add some unit tests against this function? 

I suggest a test making sure that the returned User-Agent returns a User-Agent string that meets the requirement of the API. That way if the code ever gets modified we can feel confident our code always meets the requirement of the API.

Reply:
Done. Have added a unit test case for this function. Can be found in latest commit in this PR